run_tests.py

```diff
@@ -11,6 +12,8 @@
 
 # Configure the default settings
 configure_settings()
+if django.VERSION[1] >= 7:
```

Comment:
Since we won't support less than 1.6, you won't need this `if`, just `django.setup()`